### DISCUSSION TITLE
Is Inter updated to v4 on Google Fonts?

### DISCUSSION THREAD

**Discussion by Pogimon3** [rsms/inter#653] [Q&A] [answered]
If not, how do I track the progress of Inter v4 getting on Google fonts? 

My startup uses Inter everywhere and unfortunately, many of our softwares source fonts directly from Google Fonts (e.g. Typeform, Google Slides) and there is no way to manually upload a font file.

**Comment by kenmcd** [comment, marked as answer]
No, Inter has not yet been updated to v4 in Google Fonts.

Google Fonts is tracking the issue in their tracker here:
**Update Inter to latest version #3429** 
https://github.com/google/fonts/issues/3429

So that is where you can track the progress of the update to v4 in Google Fonts.
